src/deltacode/cli.py

```diff
@@ -51,7 +51,18 @@ def write_json(deltacode, outfile, all_delta_types=False):
         ('delta_stats', deltacode.stats.to_dict()),
         ('deltas', deltas(deltacode, all_delta_types))
     ])
-
+    click.secho(get_notice(),fg="green")
+    click.echo('deltacode_options: {')
+    for option in deltacode.options:
+        click.echo("""   "{}": "{}" """.format(option,deltacode.options[option]))
+    click.echo("}")    
+    click.echo("""deltacode_version: "{}" """.format(__version__))
+    click.echo('deltacode_errors: {}'.format(collect_errors(deltacode)))
+    click.echo('deltas_count: {}'.format(len([d for d in deltas(deltacode, all_delta_types)])))
```

Comment:
I do not really care to see these in the output. #130 is relating to just the Number of Deltas and what the different factors are. 

I would like to see the counts of each particular Delta type (factors), not much else.

Reply:
Okay @majurg sir so I am updating this part